crates/trigger-http/src/lib.rs

```diff
@@ -50,6 +50,10 @@ pub struct CliArgs {
     /// The path to the certificate key to use for https, if this is not set, normal http will be used. The key should be in PKCS#8 format
     #[clap(long, env = "SPIN_TLS_KEY", requires = "tls-cert")]
     pub tls_key: Option<PathBuf>,
+
+    /// The port to listen on
+    #[clap(long = "port", env = "SPIN_HTTP_LISTEN_PORT")]
+    pub port: Option<u16>,
```

Comment:
Could you expand on how you see this interacting with `--listen`?  The way I usually override the port is e.g. `spin up --listen 127.0.0.1:4000` - is this envisaged as a shortcut to that?  Or a way to set the address and port separately?